src/Stratis.Bitcoin/Consensus/ConsensusErrors.cs

```diff
@@ -8,6 +8,7 @@ public static class ConsensusErrors
         public static readonly ConsensusError InvalidPrevTip = new ConsensusError("invalid-prev-tip", "invalid previous tip");
         public static readonly ConsensusError HighHash = new ConsensusError("high-hash", "proof of work failed");
         public static readonly ConsensusError BadCoinbaseHeight = new ConsensusError("bad-cb-height", "block height mismatch in coinbase");
+        public static readonly ConsensusError InvalidPowHeight = new ConsensusError("invalid-pow-height", "a pow block was generated passed the pow height");
```

Comment:
We use `ProofOfWorkTooHigh` in other places, perhaps use that instead.

Reply:
ah good point